src/axom/mint/docs/sphinx/sections/getting_started.rst

```diff
@@ -99,16 +159,19 @@ supplying the third argument in the ``createField()`` method invocation.
     that the number of components of the field defaults to :math:`1` and thereby
     the field is assumed to be a scalar quantity.
 
+See the :ref:`sections/tutorial` for more details on :ref:`workingWithFields`
+on a mesh.
+
 .. _step4:
 
 Step 4: Himmelblau's Function
```

Comment:
Should this be retitled to something like `Generating a scalar field` ? 
The fact that you are using Himmelblau's function seems like more of a detail.

Reply:
Yes. That's better. Thanks.